geomesa-process/pom.xml

```diff
@@ -22,12 +22,14 @@
         <dependency>
             <groupId>org.apache.accumulo</groupId>
             <artifactId>accumulo-core</artifactId>
-            <scope>provided</scope>
```

Comment:
I thought it was redundant since accumulo-core is already marked as provided in GeoMesa.